util/chplenv/compiler_utils.py

```diff
@@ -12,14 +12,29 @@
 
 
 @memoize
-def get_compiler_version(compiler):
-    version_string = '0'
+def get_compiler_name(compiler):
     if compiler == 'aarch64-gnu':
-        version_string = run_command(['aarch64-unknown-linux-gnu-gcc', '-dumpversion'])
+        return 'aarch64-unknown-linux-gnu-gcc'
     elif 'gnu' in compiler:
+        return 'gcc'
+    elif compiler.startswith('cray-prgenv-'):
```

Comment:
`compiler.startswith('cray-prgenv-'):` should go before `'gnu' in compiler:` since we want to use `cc` for cray-prgenv-gnu